django_ipam/base/models.py

```diff
@@ -0,0 +1,19 @@
+import swapper
+from django.db import models
+
+
+class AbstractSubnet(models.Model):
```

Comment:
use `openwisp_utils.base.TimeStampedEditableModel` instead of `models.Model`, like here:

https://github.com/openwisp/django-loci/blob/master/django_loci/base/models.py#L20